src/schemaToPropTypes.js

```diff
@@ -234,8 +234,8 @@ const getPropTypes = (schemaName, schema) => {
 	const requiredProps = 'required' in schema && schema.required;
 	const propTypeStringIndented = useIndentation(
 		propTypeString,
-		!!isObjectDefinition,
-		!isObjectDefinition ? false : undefined,
+		isObjectDefinition,
+		isObjectDefinition,
```

Comment:
Ehm, are you sure this is right and shouldn't be `!isObjectDefinition`?

In the previous version, when `isObjectDefinition` was true, the arguments for useIndentation were like this: `propTypeString`, `true`, `false`. Now it's `propTypeString`, `true`, `true`, which definitely is a change to the previous version of it and doesn't seem to be intended.

Reply:
Good catch, tho the logic refactor seems good now. I know the logic is now different, but indeed I only want the prefix and the suffix when `isObjectDefinition` is `true`, when it's `false` I don't want anything, otherwise I want to fallback to default, which is prefix and no suffix.